Resources/Prototypes/Body/Parts/reptilian.yml

```
@@ -36,15 +36,32 @@
+    # PIRATE END
   - type: Extractable
     juiceSolution:
       reagents:
       - ReagentId: Fat
         Quantity: 5
       - ReagentId: Blood
         Quantity: 10
+    # PIRATE START
+  - type: Tag
+    tags:
+      - Head
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)